view/frontend/layout/checkout_cart_item_renderers.xml

```diff
@@ -6,28 +6,28 @@
                 <arguments>
                     <argument name="viewModel" xsi:type="object">Extend\Integration\ViewModel\EnvironmentAndExtendStoreUuid</argument>
                 </arguments>
             </block>
         </referenceBlock>
         <referenceBlock name="checkout.cart.item.renderers.simple.actions">
-            <block class="Magento\Checkout\Block\Cart\Item\Renderer\Actions\Generic" ifconfig="extend_plans/product_protection/enable" before="-" name="extend.cart.view.type.simple.product-protection-offer" template="Extend_Integration::cart/item/renderer/actions/product-protection-simple-offer.phtml">
+            <block class="Magento\Checkout\Block\Cart\Item\Renderer\Actions\Generic" ifconfig="extend_plans/product_protection/enable_cart_offer" before="-" name="extend.cart.view.type.simple.product-protection-offer" template="Extend_Integration::cart/item/renderer/actions/product-protection-simple-offer.phtml">
                 <arguments>
                     <argument name="viewModel" xsi:type="object">Extend\Integration\ViewModel\EnvironmentAndExtendStoreUuid</argument>
                 </arguments>
             </block>
         </referenceBlock>
         <referenceBlock name="checkout.cart.item.renderers.configurable.actions">
-            <block class="Magento\Checkout\Block\Cart\Item\Renderer\Actions\Generic" ifconfig="extend_plans/product_protection/enable" before="-" name="extend.cart.view.type.configurable.product-protection-offer" template="Extend_Integration::cart/item/renderer/actions/product-protection-simple-offer.phtml">
+            <block class="Magento\Checkout\Block\Cart\Item\Renderer\Actions\Generic" ifconfig="extend_plans/product_protection/enable_cart_offer" before="-" name="extend.cart.view.type.configurable.product-protection-offer" template="Extend_Integration::cart/item/renderer/actions/product-protection-simple-offer.phtml">
                 <arguments>
                     <argument name="viewModel" xsi:type="object">Extend\Integration\ViewModel\EnvironmentAndExtendStoreUuid</argument>
                 </arguments>
             </block>
         </referenceBlock>
         <referenceBlock name="checkout.cart.item.renderers.grouped.actions">
-            <block class="Magento\Checkout\Block\Cart\Item\Renderer\Actions\Generic" ifconfig="extend_plans/product_protection/enable" before="-" name="extend.cart.view.type.grouped.product-protection-offer" template="Extend_Integration::cart/item/renderer/actions/product-protection-simple-offer.phtml">
+            <block class="Magento\Checkout\Block\Cart\Item\Renderer\Actions\Generic" ifconfig="extend_plans/product_protection/enable_cart_offer" before="-" name="extend.cart.view.type.grouped.product-protection-offer" template="Extend_Integration::cart/item/renderer/actions/product-protection-simple-offer.phtml">
```

Comment:
just changing every way that the cart offer is displayed to be conditional on the cart offer toggle (which is in turn conditional on the overall PP V2 toggle [which is in turn conditional on the `Enable Extend` toggle])

Reply:
forgot to mention: unfortunately, `ifconfig` does not support multiple conditions.